test/test_gen.py

```diff
@@ -17,6 +19,21 @@ def clean_up():
         rimraf(os.path.join('.', file + 'temp.nc'))
 
 
+def sentinel2_products_exist(path: str) -> bool:
+    """
+    Test if given *path* is likely a Sentinel-2 product path.
+
+    :param path: (directory) path
+    :return: True, if so
+    """
+    if os.path.isdir(path):
+        for fname in os.listdir(path):
+            if fname.endswith('.nc') and fname.startswith('S2'):
```

Comment:
added tiny more specific test -is it characteristic enough? or would you rather want the dataset to be opened and then checkt for some more specific things?